### PR TITLE
[Core] Referencing 'master' branch in README instead

### PR DESCRIPTION
**Pull request by Jkat**
Updating README now that 'master' is supposed to be up-to-date & 'stable'

### CODE REVIEW

**Comment by driusan** [comment]
I think we should just remove that icon.

It can only reference one branch, which means it'll get out of sync as releases get merged to different branches, and should always be "build passing" since we don't merge things unless they pass travis anyways.

**Comment by Jkat** [comment]
I think it still looks good and more professional though. It'll give the impression that our software has been tested(not that it isn't of course...................). Don't we always release with what's on the master branch? Or have we changed this?

**Comment by driusan** [comment]
That's the problem: the release is on the master branch, but the README is on all branches (referencing the master branch), so it's wrong on every other branch. So either we need to constantly be changing the READMEs on every branch every time we do a bugfix/minor/major release, or the Build Status tag will be wrong on 3/4 branches.

**Comment by driusan** [comment]
I do agree that it looks better and more professional to have it, though, I just don't know of any way we can have it so that it's not misleading.

**Comment by Jkat** [comment]
Maybe just like "master branch status: [build status tag]"

**Comment by driusan** [comment]
That seems like a reasonable compromise to me.

**Comment by Jkat** [comment]
Hmm not sure why it says I 'dismissed' @ridz1208's review..
Anyway, I made it into a table format because it looks really ugly otherwise...if anyone has a solution to format it nicer, feel free

**Comment by ridz1208** [comment]
@Jkat it dismisses automatically when changes are made

looks good to me, @driusan ??

**Comment by driusan** [comment]
Now that you've made it a table, what about keeping it in the same place as before (just below the header) and including the bugfix/minor/major branches as other rows?

**Comment by Jkat** [comment]
I added the other main branches that we care about...but I believe I have to specify the additional branches in .travis.yml.

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+branches:
+  only:
+    - master
+    - major
+    - minor
+    - bugfix
```
Comment (driusan) [inline .travis.yml]: why don't we want to run on any other branch? I think this should be a different PR if there's a reason for it (the table looks good to me now.)
